application/src/main/java/org/togetherjava/tjbot/commands/reminder/ReminderCommand.java

```diff
@@ -102,6 +121,29 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
             .insert());
     }
 
+    private void handleListCommand(SlashCommandInteractionEvent event) {
+        BiFunction<Long, Instant, String> getDescription = (channelId, remindAt) -> """
+                Channel: <#%s>
+                Remind at: <t:%s>""".formatted(channelId, remindAt.getEpochSecond());
+
+        EmbedBuilder remindersEmbed =
+                new EmbedBuilder().setTitle("Reminders").setColor(RemindRoutine.AMBIENT_COLOR);
+
+        database.read(context -> {
+            try (SelectWhereStep<PendingRemindersRecord> select =
+                    context.selectFrom(PENDING_REMINDERS)) {
+                return select
+                    .where(PENDING_REMINDERS.GUILD_ID.eq(event.getGuild().getIdLong())
+                        .and(PENDING_REMINDERS.AUTHOR_ID.eq(event.getUser().getIdLong())))
+                    .fetch();
```

Comment:
u might want to sort them ascending by date. so that it displays the soonest reminders first

Reply:
```java
.sortAsc(PENDING_REMINDERS.REMIND_AT)
```
like this??

Reply:
I believe it is `.orderBy(PENDING_REMIMDERS.REMIND_AT.asc())`

Reply:
If i got it right sortAsc applys after getting the result and orderBy affects the sql query. If i had to choose i would use orderBy because the database is probably faster at ordering before fetching than you doing it yourself after fetching.

Reply:
now it sorts ascending by creating date. i meant more like ascending on when its fired. so that u can see them in the order they will be fired. @Taz03